src/main/java/uk/nhs/tis/trainee/notifications/service/ProgrammeMembershipService.java

```diff
@@ -262,6 +267,10 @@ private void createInAppNotifications(ProgrammeMembership programmeMembership,
           ltftVersion, Map.of(
               LOCAL_OFFICE_CONTACT_FIELD, localOfficeContact,
               LOCAL_OFFICE_CONTACT_TYPE_FIELD, localOfficeContactType));
+      createUniqueInAppNotification(programmeMembership, notificationsAlreadySent, DEFERRAL,
+          deferralVersion, Map.of(
+              LOCAL_OFFICE_CONTACT_FIELD, localOfficeContact,
+              LOCAL_OFFICE_CONTACT_TYPE_FIELD, localOfficeContactType));
```

Comment:
This map is the same as above, so it could be extracted in to a variable to be passed in both calls.

Reply:
Just wondering don't we need to get the localOfficeContact and localOfficeContactType values according to `DEFERRAL` type instead of `LTFT`?

Reply:
Ah, you're right @DorisWongNHS good spot!